src/sentry/static/sentry/app/views/settings/projectIncidentRules/ruleForm.tsx

```diff
@@ -44,6 +47,42 @@ type AlertRuleThresholdKey = {
   [AlertRuleThreshold.RESOLUTION]: 'resolveThreshold';
 };
 
+enum TimeWindow {
+  ONE_MINUTE = 60,
+  FIVE_MINUTES = 300,
+  TEN_MINUTES = 600,
+  FIFTEEN_MINUTES = 900,
+  THIRTY_MINUTES = 1800,
+  ONE_HOUR = 3600,
+  TWO_HOURS = 7200,
+  FOUR_HOURS = 14400,
+  ONE_DAY = 86400,
+}
+
+const TIME_WINDOW_MAP: Map<TimeWindow, string> = new Map([
+  [TimeWindow.ONE_MINUTE, t('1 minute')],
+  [TimeWindow.FIVE_MINUTES, t('5 minutes')],
+  [TimeWindow.TEN_MINUTES, t('10 minutes')],
+  [TimeWindow.FIFTEEN_MINUTES, t('15 minutes')],
+  [TimeWindow.THIRTY_MINUTES, t('30 minutes')],
+  [TimeWindow.ONE_HOUR, t('1 hour')],
+  [TimeWindow.TWO_HOURS, t('2 hours')],
+  [TimeWindow.FOUR_HOURS, t('4 hours')],
+  [TimeWindow.ONE_DAY, t('24 hours')],
+]);
+
+const TIME_WINDOW_TO_PERIOD: Map<number, string> = new Map([
```

Comment:
Why is this map `<number, string>` but the one on line 62 is `<TimeWindow, string>`?